bin/thinkingThingsClient

```diff
@@ -214,6 +261,11 @@ var commands = {
         description: '\tSend a new gps measure',
         handler: gpsLocation
     },
+    'gsm': {
+        parameters: ['mcc', 'mnc', 'cell-id', 'lac', 'dbm'],
+        description: '\tSend a new gps measure',
```

Comment:
LGTM apart from a super-minor comment :) Once it is decided upon, I'll land the PR... :)